experimental/CHANGELOG.md

```diff
@@ -88,6 +88,7 @@ All notable changes to experimental packages in this project will be documented
 
 ### :rocket: (Enhancement)
 
+* feat(instrumentation): Add allowUrls config option to web instrumentation [#4938](https://github.com/open-telemetry/opentelemetry-js/pull/4938) @jairo-mendoza
```

Comment:
oops, this has moved down due to the recent release - please move it up to unreleased 🙂